AWS/architecture/database/tables.sql

```diff
@@ -0,0 +1,71 @@
+DROP TABLE IF EXISTS artist_popularity;
+DROP TABLE IF EXISTS track_genre;
+DROP TABLE IF EXISTS track_popularity;
+DROP TABLE IF EXISTS track_artist;
+DROP TABLE IF EXISTS artist;
+DROP TABLE IF EXISTS genre;
+DROP TABLE IF EXISTS track;
+
+CREATE TABLE IF NOT EXISTS track (
+    track_id INT GENERATED ALWAYS AS IDENTITY,
+    track_name VARCHAR(200),
+    track_danceability INT NOT NULL,
+    track_energy INT NOT NULL,
+    track_valence INT NOT NULL,
+    track_tempo INT NOT NULL,
+    track_speechiness INT NOT NULL,
+    in_spotify BOOLEAN NOT NULL,
+    in_tiktok BOOLEAN NOT NULL DEFAULT false,
+    tiktok_rank INT,
+    spotify_rank INT,
+    spotify_id VARCHAR(200),
+    PRIMARY KEY(track_id)
+);
+
+CREATE TABLE IF NOT EXISTS artist (
+    artist_id INT GENERATED ALWAYS AS IDENTITY,
+    spotify_name VARCHAR(100),
+    PRIMARY KEY(artist_id)
+);
+
+CREATE TABLE IF NOT EXISTS artist_popularity (
+    popularity_id INT GENERATED ALWAYS AS IDENTITY,
+    artist_id INT NOT NULL,
+    artist_popularity INT NOT NULL,
+    follower_count INT NOT NULL,
+    recording_at TIMESTAMP NOT NULL,
```

Comment:
This might be better as recorded_at, and also would be useful to have a default of CURRENT_TIMESTAMP so we don't need to add it in manually